src/NLU.DevOps.ModelPerformance/NLUAccuracy.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace NLU.DevOps.ModelPerformance
+{
+    using ConsoleTables;
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text;
+
+    /// <summary>
+    /// This class contains utils function for performance calculations
+    /// and printing
+    /// </summary>
+    public static class NLUAccuracy
+    {
+        /// <summary>
+        /// Calculates the precision from a confusion matrix
+        /// </summary>
+        /// <param name="cm">confusion matrix metrics</param>
+        /// <returns>The precision result</returns>
+        public static double Precision(this ConfusionMatrix cm)
+        {
+            return Divide(cm.TruePositive, cm.TruePositive + cm.FalsePositive);
+        }
+
+        /// <summary>
+        ///  Calculates the recall from a confusion matrix
+        /// </summary>
+        /// <param name="cm"> confusin matrix metrics</param>
+        /// <returns> The recall result</returns>
+        public static double Recall(this ConfusionMatrix cm)
+        {
+            return Divide(cm.TruePositive, cm.TruePositive + cm.FalseNegative);
+        }
+
+        /// <summary>
+        /// Calculates the f1 score from a confusion matrix
+        /// </summary>
+        /// <param name="cm"> confusion matrix metrics</param>
+        /// <returns> the f1 result</returns>
+        public static double F1(ConfusionMatrix cm)
```

Comment:
Seems to be a mix of extension and non-extension, any reason for that?

Reply:
no by mistake. fixed it